python/packages/autogen-core/tests/test_utils/__init__.py

```diff
@@ -46,3 +55,31 @@ def __init__(self) -> None:
 
     async def on_message(self, message: Any, ctx: MessageContext) -> Any:
         raise NotImplementedError
+
+
+@default_subscription
+class MyAgent(RoutedAgent):
```

Comment:
Where is `MyAgent` used?

Reply:
Cleaned it up, it was introduced in another PR of mine.